templates/example_visualization.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<p>This looks much better! I think it would be good to define this visualization folder at the top, next to data_dir. It also makes the most sense I think to have this be inside of the data_dir, rather than floating off on its own. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>